### PR TITLE
[AC-533] added first time intro slides to app

### PR DESCRIPTION
**Pull request by rishabh-997**
## Description of what I changed
I used AppIntro library to produce intro slides and used SharedPreference to show it only once.

## Issue I worked on

JIRA Issue: https://issues.openmrs.org/browse/AC-533

## Checklist: I completed these to help reviewers :)

- [x] My pull request only contains **ONE single commit**
(the number above, next to the 'Commits' tab is 1).

- [ ] I have **added tests** to cover my changes. (If you refactored
existing code that was well tested you do not have to add tests)

- [x] All new and existing **tests passed**.

- [x] My pull request is **based on the latest changes** of the master branch.

### CODE REVIEW

**Comment by rishabh-997** [comment]
@f4ww4z i have added the introduction slides with proper material UI. Review it plzz

**Comment by PrathameshMutkure** [comment]
Hii @rishabh-997, can you have a look at [OpenMRS PR Tips](https://github.com/openmrs/openmrs-contrib-android-client/pull/627) and add Ticket ID to you title?

**Comment by rishabh-997** [comment]
> Hii @rishabh-997, can you have a look at [OpenMRS PR Tips](https://github.com/openmrs/openmrs-contrib-android-client/pull/627) and add Ticket ID to you title :)

Can you tell me the mistake specifically sir, i made sure i did it within guidelines

**Comment by PrathameshMutkure** [comment]
You didn't included the ticket ID in it, it's okay now :)

**Comment by rishabh-997** [comment]
Can you please review this PR @f4ww4z

**Comment by rishabh-997** [comment]
Can you review or suggest changes to it @f4ww4z ?? I have submitted the code review too

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/Introduction/IntroActivity.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    protected OpenMRS mOpenMRS = OpenMRS.getInstance();
+
+    @Override
+    protected void onCreate(@Nullable Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+
+        addSlide(AppIntroFragment.newInstance("Welcome", "The global OpenMRS community works together to build the world's leading open source enterprise electronic medical record system platform.", R.drawable.openmrs_action_logo, Color.parseColor("#5B57A6")));
+        addSlide(AppIntroFragment.newInstance("Register Patients", "You can register your new patients to our server", R.drawable.ico_registry, Color.parseColor("#F26522")));
+        addSlide(AppIntroFragment.newInstance("Find Patients", "You can find and monitor those patients", R.drawable.ico_search, Color.parseColor("#009384")));
+        addSlide(AppIntroFragment.newInstance("Monitor Visits", "We help you monitor patient's visits too... Exciting ??? Lets get started", R.drawable.ico_visits, Color.parseColor("#F0A815")));
```
[the remark this answers is not in the document]
Comment (f4ww4z) [reply openmrs-client/src/main/java/org/openmrs/mobile/activities/Introduction/IntroActivity.java]: Also add a slide for the manage providers, location button and settings page.

[the remark this answers is not in the document]
Comment (rishabh-997) [reply openmrs-client/src/main/java/org/openmrs/mobile/activities/Introduction/IntroActivity.java]: okay sir, on it

**Comment by f4ww4z** [comment]
@rishabh-997 You need to pull the latest changes from master first, before comitting your own. That will resolve the merge conflicts. Use something like `git pull --rebase upstream master`

File: openmrs-client/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -401,4 +401,22 @@
...
+    <!-- introduction Activity -->
+    <string name="intro_welcome">Welcome</string>
+    <string name="intro_register">Register Patients</string>
+    <string name="intro_find">Find Patients</string>
+    <string name="intro_monitor">Monitor Visits</string>
+    <string name="intro_manage">Manage Providers</string>
+    <string name="intro_location">Location</string>
+    <string name="intro_settings">Settings</string>
+
+    <string name="intro_welcome_desc">The global OpenMRS community works together to build the world leading open source enterprise electronic medical record system platform.</string>
```
Comment (f4ww4z) [inline openmrs-client/src/main/res/values/strings.xml]:
```suggestion
    <string name="intro_welcome_desc">The global OpenMRS community works together to build the world's leading open source electronic medical record system.</string>
```

File: openmrs-client/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -401,4 +401,22 @@
...
+    <string name="intro_register_desc">You can register your new patients to our server</string>
+    <string name="intro_find_desc">You can find and monitor those patients</string>
+    <string name="intro_monitor_desc">We help you monitor patient visits too.</string>
```
Comment (f4ww4z) [inline openmrs-client/src/main/res/values/strings.xml]:
```suggestion
    <string name="intro_monitor_desc">We help you monitor patient visits too</string>
```

File: openmrs-client/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -401,4 +401,22 @@
...
+    <string name="intro_manage_desc">You can manage providers using our app</string>
+    <string name="intro_location_desc">You can change your location at a simple click</string>
+    <string name="intro_settings_desc">Try out our dark mode feature. Exciting ??? Lets get started</string>
```
Comment (f4ww4z) [inline openmrs-client/src/main/res/values/strings.xml]:
```suggestion
    <string name="intro_settings_desc">Try out our dark mode feature.\nExciting? Let's get started.</string>
```

**Comment by rishabh-997** [comment]
Since the white background was required, I changed the text colour to black...

Intro Page with Logo.jpeg?alt=media&token=b8ede8dc-66d2-4ca2-ab15-b8feb2eb45be)

File: openmrs-client/src/main/res/values/strings.xml [hunk 5]
```diff
@@ -401,4 +401,22 @@
...
+    <string name="intro_welcome">Welcome</string>
+    <string name="intro_register">Register Patients</string>
+    <string name="intro_find">Find Patients</string>
+    <string name="intro_monitor">Monitor Visits</string>
+    <string name="intro_manage">Manage Providers</string>
+    <string name="intro_location">Location</string>
+    <string name="intro_settings">Settings</string>
+
+    <string name="intro_welcome_desc">The global OpenMRS community works together to build the world\'s leading open source electronic medical record system.</string>
+    <string name="intro_register_desc">You can register your new patients to our server</string>
```
Comment (f4ww4z) [inline openmrs-client/src/main/res/values/strings.xml]:
```suggestion
    <string name="intro_register_desc">Register new patients to our server.</string>
```

File: openmrs-client/src/main/res/values/strings.xml [hunk 6]
```diff
@@ -401,4 +401,22 @@
...
+    <string name="intro_find_desc">You can find and monitor those patients</string>
```
Comment (f4ww4z) [inline openmrs-client/src/main/res/values/strings.xml]:
```suggestion
    <string name="intro_find_desc">You can find and monitor those patients.</string>
```

File: openmrs-client/src/main/res/values/strings.xml [hunk 7]
```diff
@@ -401,4 +401,22 @@
...
+    <string name="intro_monitor_desc">We help you monitor patient visits too</string>
```
Comment (f4ww4z) [inline openmrs-client/src/main/res/values/strings.xml]:
```suggestion
    <string name="intro_monitor_desc">We help you monitor patient visits too.</string>
```

File: openmrs-client/src/main/res/values/strings.xml [hunk 8]
```diff
@@ -401,4 +401,22 @@
...
+    <string name="intro_manage_desc">You can manage providers using our app</string>
```
Comment (f4ww4z) [inline openmrs-client/src/main/res/values/strings.xml]:
```suggestion
    <string name="intro_manage_desc">You can manage providers using our app.</string>
```

File: openmrs-client/src/main/res/values/strings.xml [hunk 9]
```diff
@@ -401,4 +401,22 @@
...
+    <string name="intro_location_desc">You can change your location at a simple click</string>
```
Comment (f4ww4z) [inline openmrs-client/src/main/res/values/strings.xml]:
```suggestion
    <string name="intro_location_desc">You can change your location at a simple click.</string>
```

**Pull request merged**